resources/visualizations/Application-Tags-Summary.json

```diff
@@ -0,0 +1,9 @@
+{
+   "visState": "{\"type\":\"table\",\"params\":{\"perPage\":10,\"showPartialRows\":false,\"showMeticsAtAllLevels\":false},\"aggs\":[{\"id\":\"1\",\"type\":\"sum\",\"schema\":\"metric\",\"params\":{\"field\":\"TotalBytes\"}},{\"id\":\"2\",\"type\":\"terms\",\"schema\":\"bucket\",\"params\":{\"field\":\"ApplicationTags\",\"size\":50,\"order\":\"desc\",\"orderBy\":\"1\"}},{\"id\":\"3\",\"type\":\"sum\",\"schema\":\"metric\",\"params\":{\"field\":\"TotalPackets\"}}],\"listeners\":{}}", 
```

Comment:
For this Metric, we want to use `Sum of TotalBytesDelta` and `Sum of TotalPacketsDelta` to specifically only calculate these metrics over the course of the time window, rather than ALL reports for sessions that were updated in this time window.

I spoke to @lmayron about this over Slack and he said that is the behavior we are trying to display here.